Automattic/jetpack
# actions: Reenable experimental PHP 8.1 run

#### Changes proposed in this Pull Request:

This will reenable tests against PHP 8.1, which had been disabled in #22850.

#### Jetpack product discussion


p1644842786633359-slack-CBG1CP4EN
p9dueE-4qH-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure that the PHP 8.1 tests are either passing or are failing because of errors in our own code rather than WordPress Core.
* Or make sure GitHub Actions will now show the failing test with some sort of "expected" status, as described in [actions/toolkit#399](https://href.li?https://github.com/actions/toolkit/issues/399), rather than various status indicators showing "failing".
